shanoir-ng-datasets/src/main/java/org/shanoir/ng/datasetfile/DatasetFileApiController.java

```diff
@@ -0,0 +1,109 @@
+package org.shanoir.ng.datasetfile;
+
+import java.io.File;
+import java.time.LocalDateTime;
+import javax.validation.Valid;
+
+import org.apache.commons.io.FileUtils;
+import org.shanoir.ng.datasetfile.service.DatasetFileApi;
+import org.shanoir.ng.datasetfile.service.DatasetFileService;
+import org.shanoir.ng.shared.exception.ErrorModel;
+import org.shanoir.ng.shared.exception.RestServiceException;
+import org.shanoir.ng.shared.service.DicomServiceApi;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.multipart.MultipartFile;
+
+import io.swagger.annotations.ApiParam;
+
+@Controller
+public class DatasetFileApiController implements DatasetFileApi {
+
+	@Autowired
+	DatasetFileService datasetFileService;
+
+	@Autowired
+	@Qualifier("stowrs")
+	DicomServiceApi stowRsService;
+
+	@Autowired
+	@Qualifier("cstore")
+	DicomServiceApi cStoreService;
```

Comment:
please make private to really hide the attribute, encapsulation pattern, same for sFService + stowRsService above